x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -60,18 +60,9 @@ const TitleText = styled.span`
   margin-right: 12px;
 `;
 
-const StyledEuiPanel = styled(EuiPanel)<{ $isFullScreen: boolean }>`
-  display: flex;
-  flex-direction: column;
-
-  ${({ $isFullScreen }) =>
-    $isFullScreen &&
-    `
-      border: 0;
-      box-shadow: none;
-      padding-top: 0;
-      padding-bottom: 0;
-  `}
+const AlertsTableWrapper = styled.div`
```

Comment:
also might be worth noting that tGrid runs in 2 "modes": integrated, where the redux actions/state are used within a bigger app like security solution, or in standalone, where all of those things are run in a self contained way. Tldr if you want these styling changes to be reflected in security solution as well, need to make these same changes https://github.com/elastic/kibana/blob/master/x-pack/plugins/timelines/public/components/t_grid/integrated/index.tsx#L68

Reply:
The integrated file was updated, but these look to be setup slightly differently: https://github.com/elastic/kibana/pull/108903

Reply:
in my humble opinion, I feel like it is better to remove the the `EuiPanel` and just use a div than disabling everything from this component.